src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((declaration) => declaration.trim().length > 0)
+    .map((declaration) => {
+      const [property, value] = declaration
+        .split(':')
+        .map((item) => item.trim());
+
+      // Залишити властивість з дефісами
+      return { [property]: value };
+    })
+    .reduce((accumulator, current) => Object.assign(accumulator, current), {});
```

Comment:
Using Object.assign in reduce is not the most efficient way to build an object from an array of key-value pairs. Consider using the spread operator or directly assigning the property to the accumulator object.